hyperopt/spark.py

```diff
@@ -6,12 +6,14 @@
 import time
 import timeit
 
-import cloudpickle
-
 from hyperopt import base, fmin, Trials
 from hyperopt.utils import coarse_utcnow, _get_logger, _get_random_id
-from pyspark.sql import SparkSession
 
+try:
```

Comment:
I had to do this in order to run other tests without Spark being available.  I added a test case for this.